central/blob/datastore/store/postgres/gen.go

```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.Blob
```

Comment:
Based on how you are using it from the datastore, would it make since to simplify it to the singleton generation with `--singleton` instead of generating a full store?

Reply:
This is not a singleton. It may contains multiple entities.